app/services/RuleProvisionerService.scala

```diff
@@ -13,11 +13,14 @@ import rules.BucketRuleManager
 import matchers.LanguageToolFactory
 import model.LTRule
 import model.LTRuleXML
+import utils.CloudWatchClient
+import utils.Metrics
 
 class RuleProvisionerService(
   bucketRuleManager: BucketRuleManager,
   matcherPool: MatcherPool,
-  languageToolFactory: LanguageToolFactory
+  languageToolFactory: LanguageToolFactory,
+  cloudWatchClient: CloudWatchClient
```

Comment:
There may be a nice way of providing access to the cloudwatch client, incase we would like to use it on other places. @sihil